### PR TITLE
Consolidate PRs from Fabio (SOFTWARE-3470)

### PR DESCRIPTION
**Pull request by edquist**
This pulls in #2755, #2756, #2757, #2758, #2762 & #2764 (from @biozit) so we don't have to hit too many buttons and wait too many times.

@djw8605 including you since you left some comments about changing the directory path to NRP.

### CODE REVIEW

**Comment by edquist** [comment]
I added some misc cleanups, renames, missing IDs, and missing FACILITY.yaml ... Looks like the checks are passing now!

File: topology/Massachusetts Green High Performance Computing Center/NRP/MGHPCCCachingInfrastructure.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Resources:
+  MGHPCC_NRP_OSDF_CACHE:
+    Active: false
+    Description: MGHPCC NRP OSDF Cache
+    ID: 1348
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Fabio Andrijauskas
+          ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
```
Comment (djw8605) [inline topology/Massachusetts Green High Performance Computing Center/NRP/MGHPCCCachingInfrastructure.yaml:15]: We should have a backup.  How about Huijun?

File: topology/University of Hawaii/Information Technology Center/ITC-PRP.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+GroupID: 1319
```
Comment (djw8605) [inline topology/University of Hawaii/Information Technology Center/ITC-PRP.yaml:1]: PRP is no more, long live NRP.

The filename should be NRP, PRP is no longer a thing as of October. 1.

**Review by djw8605 — CHANGES_REQUESTED** [review]
Some comments.

I don't mind combining these pull requests, but they are logically separate.  One is adding a NRP cache to MGHPCC, the other is adding an origin (unrelated to NRP) to Hawaii, and creating sites and VO.

File: topology/University of Hawaii/Information Technology Center/ITC-PRP.yaml (hunk 2 above)
Comment (edquist) [reply topology/University of Hawaii/Information Technology Center/ITC-PRP.yaml]: Ah good catch; i had only changed the dir name before.  Updated.

File: topology/Massachusetts Green High Performance Computing Center/NRP/MGHPCCCachingInfrastructure.yaml (hunk 1 above)
Comment (edquist) [reply topology/Massachusetts Green High Performance Computing Center/NRP/MGHPCCCachingInfrastructure.yaml:14]: Added (in each new resource)

**Comment by edquist** [comment]
>I don't mind combining these pull requests, but they are logically separate. One is adding a NRP cache to MGHPCC, the other is adding an origin (unrelated to NRP) to Hawaii, and creating sites and VO.

Makes sense - in that case it might look nicer to break this PR into two separate octopus PRs that pull in and tidy up Fabio's PRs.  If you want I can do that, though in that case I won't get to it today.

**Review by djw8605 — CHANGES_REQUESTED** [review]
Sorry, Huijun should only be added to the cache node, not the Hawaii.

**Review by biozit — APPROVED** [review]

**Comment by biozit** [comment]
Hi all,
  
 Please, should I do something else to finish this?

Thank you.

**Comment by edquist** [comment]
>Sorry, Huijun should only be added to the cache node, not the Hawaii.

Thanks @djw8605 for catching this - I've removed him from Hawaii now.

**Comment by edquist** [comment]
Hi @biozit, 

> should I do something else to finish this?

I think we've got everything in here now, and if we get an approving review (from Derek or Brian) then we can merge this PR.

In the future it will be better to put all the logically related changes into a single PR (this one probably should have been two), but to try to simplify things this time around I merged your #2755, #2756, #2757, #2758, #2762 & #2764 into this PR, so that once this one is merged the rest of those will be included and marked closed automatically.

Anyway sorry for the delays - we're almost to the finish line here :)

**Review by djw8605 — APPROVED** [review]
LGTM

**Comment by djw8605** [comment]
Indeed, this is logically 2 separate things, and should have been 2 pull requests, but it's fine for now.

**Pull request merged**